packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html

```diff
@@ -108,6 +107,13 @@
                 </oui-tile-description>
                 <oui-action-menu data-compact data-placement="end">
                     <oui-action-menu-item
+                        data-ng-if="$ctrl.ola.isConfigured()"
+                        data-ui-sref="app.dedicated.server.interfaces.ola-reset"
```

Comment:
Could we prefer using a `href` instead of `ui-sref`

Reply:
why?

Reply:
This way we avoid strong binding between ui-router and the view and we can also extract this as a component more easily

Reply:
did you see that there is already like 8 other ui-sref in this template ? ^^

Reply:
@frenautvh yes I'm aware we have to deal with legacy code but if we could apply some of those practices when adding new code, it's not perfect but it's a beginning :)